www/site/content/docs/chef-workstation/getting-started.md

```diff
@@ -33,52 +33,60 @@ inspec version: 2.1.72
 
 ## Ad-hoc remote execution with `chef-run`
 
-The `chef-run` utility allows you to execute ad-hoc configuration updates on the systems you manage without needing to first set up a Chef server. With chef-run, you connect to servers over SSH or WinRM, and can apply single resources, recipes, or entire cookbooks directly from your local workstation.
-
+The `chef-run` utility allows you to execute ad-hoc configuration updates on the systems you manage without setting up a Chef server. With `chef-run`, you connect to servers over SSH or WinRM, and you can apply single resources, recipes, or even entire cookbooks directly from the command line.
 
 ### Example: Installing NTP Server
 
-Chef Workstation combines the power of InSpec and chef-run to give you the ability to easily detect and correct issues on any target instance. A common task that an environment maintainer performs is ensuring that the Network Time Protocol (NTP) is installed, so clocks are kept in sync between servers. InSpec allows us to simply query whether the package is installed via its package resource:
+Chef Workstation combines the power of InSpec and `chef-run`, giving you the ability to easily detect and correct issues on any target instance. One common task that administrators perform in their environments is installing the Network Time Protocol (NTP), which keeps the clocks in sync between servers. InSpec allows us to check if the package is installed with a simple query, using the InSpec `package` resource:
 
 ```ruby
 describe package('ntp') do
   it { should be_installed }
 end
  ```
 
-Chef provides a similar single-resource solution for ensuring the package is installed:
+Chef also provides a single-resource solution to install the Network Time Protocol package:
 
 ```ruby
 package 'ntp' do
   action :install
 end
 ```
 
-Use chef-run, to converge targets against a single resource without needing to create a cookbook or recipe -- run the resource directly from the command-line:
+With `chef-run`, you can run the resource directly from the command-line, converging your targets with a single resource, without creating a cookbook or recipe:
 
 ```bash
 chef-run myhost package ntp action=install
 ```
 
-Combined with the InSpec resource to validate whether the package was installed successfully, we have everything we need to define our requirements, and make sure they're met with two simple commands.
+Combined with executing an InSpec scan to validate if the package was successfully installed, we have everything we need to define our requirements, and make sure they're met with two simple commands, either locally or remotely.
```

Comment:
Suggest: "scan to validate successful package installation" instead of "validate if the package was successfully installed".